reference/promotions.v3.yml

```diff
@@ -1642,6 +1752,36 @@ components:
               total: 5
               success: 3
               failed: 2
+
+    BulkCouponCodesResponse:
```

Comment:
why we have a duplicate definition here ? 🤔

Reply:
if this is just the response for the endpoint, maybe consider inline it instead of defining it as a component ? is it going to be re-used elsewhere ?